README.md

```diff
@@ -1,46 +1,50 @@
-# Home assistant Sun card
-Home assistant Sun card based on Google weather design
+
+# Lovelace Sun Card: Google Weather-Inspired Design
```

Comment:
Lovelace as a word is unnecessary here. A few months ago the Home Assistant core team decided to advertise the name "lovelace" and talk about the Home Assistant UI instead. I can't remember whether this was in the release notes or somewhere in the community.

Suggest "Sun Card: A _Google Weather_ Inspired Component for Home Assistant"